lib/jni/LogManager_jni.cpp

```diff
@@ -10,12 +10,12 @@
 #endif
 #endif
 
+#include <utils/Utils.hpp>
```

Comment:
Very minor nit: Edge team usually reserved the `<...>` for system headers. Thus, we usually have the `"..."` elsewhere. But I do not personally have a strong opinion on enforcing this stylistic requirement.